library/analytics/src/main/java/org/cru/godtools/analytics/AnalyticsService.java

```diff
@@ -65,9 +76,13 @@
     public static final String CATEGORY_CONTENT_EVENT = "Content Event";
 
     private Tracker mTracker = null;
+    private com.snowplowanalytics.snowplow.tracker.Tracker mSnowPlowTracker = null;
 
     /* Adobe Analytics */
     private final Executor mAdobeAnalyticsExecutor = Executors.newSingleThreadExecutor();
+    /* SnowPlow */
+    private final Executor mSnowPlowExecutor = Executors.newSingleThreadExecutor();
```

Comment:
I'd be fine with using the same executor for both Adobe Analytics and SnowPlow. the main purpose of the executor is to just push processing off of the main thread and to serialize the events it's tracking.